server/routes.js

```diff
@@ -0,0 +1,9 @@
+var controller = require('../api/controller.js');
+var router = require('express').Router();
+
+router.get('/product', controller.product.get);
```

Comment:
Following this through it's a get all products. same for the endpoints below, they operate not on 1 product but on all productS. Consider renaming.

Also, I realize that you have only hundreds of records in your db right now, but writing a "get all the things" route is dangerous, add a default limit (can be 1000), and consider adding a query param for changing the limit as a stretch

Reply:
This is just a first stab on the /productS route.  WIP on 'feature-api' branch.